flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/legacy/metrics/AbstractMetricsHandler.java

```diff
@@ -39,12 +39,15 @@
  * <p>If the query parameters do not contain a "get" parameter the list of all metrics is returned.
  * {@code [ { "id" : "X" } ] }
  *
- * <p>If the query parameters do contain a "get" parameter a comma-separate list of metric names is expected as a value.
- * {@code /get?X,Y}
+ * <p>If the query parameters do contain a "get" parameter, a comma-separated list of metric names is expected as a value.
+ * {@code /metrics?get=X,Y}
  * The handler will then return a list containing the values of the requested metrics.
  * {@code [ { "id" : "X", "value" : "S" }, { "id" : "Y", "value" : "T" } ] }
  */
 public abstract class AbstractMetricsHandler extends AbstractJsonRequestHandler {
+
```

Comment:
Why not putting all these (`PARAMETER_METRICS`, `PARAMETER_AGGREGATION`, `SUBTASK_METRICS_REST_PATH` even `"/jobs/metrics"`, etc) in a `MetricsURLConstants` class (or sth like this) so that there is a central place for all? This could help maintenance.

Reply:
This will be resolved once we port it to the new REST stuff anyway, as they will then reside in the repsective parameter/header classes.